conf/application.conf

```diff
@@ -19,6 +19,7 @@ application.router=prod.Routes
 application.global=uk.gov.hmrc.nisp.config.ApplicationGlobal
 
 application.session.httpOnly=true
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 www.google-analytics.com data:"
```

Comment:
The ports specified are for files that are served as part of the HTML page. Things like javascript and css files for example. Anything else doesn't have any impact on the security header.

As @andrewgee mentioned, PTA and other ports are only in the backend.